treeherder/model/fixtures/repository.json

```diff
@@ -1036,7 +1036,7 @@
         "name": "example-addon",
         "url": "https://github.com/mozilla/example-addon-repo",
         "branch": "master",
-        "active_status": "active",
+        "active_status": "onhold",
```

Comment:
I didn't add this one either, but I would like it to be left enabled as I may work on it this quarter.